static/app/plugins/components/pluginIcon.tsx

```diff
@@ -118,6 +119,7 @@ export const ICON_PATHS = {
   victorops,
   zepel,
   zulip,
+  notion,
```

Comment:
Move `notion` import to below `netlify` because the list is alphabetically ordered